src/main/resources/application.properties

```
@@ -1,4 +1,5 @@
 handlebars.suffix=.html
 handlebars.enabled=true
 spring.jpa.properties.hibernate.show_sql=true
-spring.jpa.properties.hibernate.format_sql=true
\ No newline at end of file
+spring.jpa.properties.hibernate.format_sql=true
+server.port = 8082
```

```suggestion
server.port = 8082

```

EOF를 추가해주세요~

- 여러 버전의 유닉스 OS간의 공통된 API 및 인터페이스를 정의해놓은 POSIX 명세 중 text file 부분을 보면 유닉스 시스템은 text file들을 서로 구분할때 EOL을 사용하기 때문에 파일을 만들때마다 파일의 끝에 EOL을 넣어주는 것이 안전하다.

---

몰랐던 내용이네요! 추가했습니다